### ISSUE TITLE
[assemble] Jdeps fails to find module org.reactivestreams for a Quarkus app

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1055]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Clone https://github.com/aalmiray/q-cli-jlink at [e659416](https://github.com/aalmiray/q-cli-jlink/commit/e6594165884a6c322f13c7fc15963e425c580b22)
2. `mvn -Passemble package`

### Expected Behaviour

A custom Java Runtime successfully created at `target/jreleaser/assemble/q-cli/jlink`

### Actual Behaviour

Assembly error due to missing module

```
jdeps --multi-release base --ignore-missing-deps --print-module-deps universal/io.quarkus.quarkus-fs-util-0.0.9.jar universal/org.jboss.logmanager.jboss-logmanager-embedded-1.0.11.jar universal/io.smallrye.common.smallrye-common-constraint-1.13.2.jar universal/org.reactivestreams.reactive-streams-1.0.3.jar universal/org.eclipse.microprofile.config.microprofile-config-api-2.0.1.jar universal/jakarta.annotation.jakarta.annotation-api-1.3.5.jar universal/io.smallrye.reactive.mutiny-1.7.0.jar universal/org.graalvm.sdk.graal-sdk-22.2.0.jar universal/jakarta.el.jakarta.el-api-3.0.3.jar universal/info.picocli.picocli-4.6.3.jar universal/io.smallrye.config.smallrye-config-common-2.12.1.jar universal/org.eclipse.microprofile.context-propagation.microprofile-context-propagation-api-1.2.jar universal/quarkus-run.jar universal/org.jboss.slf4j.slf4j-jboss-logmanager-1.2.0.Final.jar universal/io.quarkus.quarkus-arc-2.14.0.Final.jar universal/jakarta.interceptor.jakarta.interceptor-api-1.2.5.jar universal/org.slf4j.slf4j-api-1.7.36.jar universal/io.smallrye.common.smallrye-common-classloader-1.13.2.jar universal/io.smallrye.config.smallrye-config-core-2.12.1.jar universal/io.smallrye.common.smallrye-common-io-1.13.2.jar universal/io.smallrye.config.smallrye-config-2.12.1.jar universal/jakarta.transaction.jakarta.transaction-api-1.3.3.jar universal/org.wildfly.common.wildfly-common-1.5.4.Final-format-001.jar universal/io.quarkus.quarkus-development-mode-spi-2.14.0.Final.jar universal/io.quarkus.arc.arc-2.14.0.Final.jar universal/io.smallrye.common.smallrye-common-annotation-1.13.2.jar universal/jakarta.enterprise.jakarta.enterprise.cdi-api-2.0.2.jar universal/io.quarkus.quarkus-picocli-2.14.0.Final.jar universal/io.github.crac.org-crac-0.1.1.jar universal/io.smallrye.common.smallrye-common-function-1.13.2.jar universal/io.quarkus.quarkus-core-2.14.0.Final.jar universal/org.jboss.threads.jboss-threads-3.4.3.Final.jar universal/io.quarkus.quarkus-bootstrap-runner-2.14.0.Final.jar universal/org.jboss.logging.jboss-logging-3.5.0.Final.jar universal/org.jboss.logging.jboss-logging-annotations-2.2.1.Final.jar universal/io.smallrye.common.smallrye-common-expression-1.13.2.jar universal/jakarta.inject.jakarta.inject-api-1.0.jar
Exception in thread "main" java.lang.module.FindException: Module org.reactivestreams not found, required by io.smallrye.mutiny
	at java.base/java.lang.module.Resolver.findFail(Resolver.java:877)
	at java.base/java.lang.module.Resolver.resolve(Resolver.java:191)
	at java.base/java.lang.module.Resolver.resolve(Resolver.java:140)
	at java.base/java.lang.module.Configuration.resolve(Configuration.java:422)
	at java.base/java.lang.module.Configuration.resolve(Configuration.java:256)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsConfiguration$Builder.build(JdepsConfiguration.java:564)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.buildConfig(JdepsTask.java:603)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.run(JdepsTask.java:557)
	at jdk.jdeps/com.sun.tools.jdeps.JdepsTask.run(JdepsTask.java:533)
	at jdk.jdeps/com.sun.tools.jdeps.Main.main(Main.java:49)
```

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.3.1
- **JDK Version:** 11

**Comment by aalmiray** [comment]
The following command successfully produces a list of modules

```
jdeps --multi-release base --ignore-missing-deps --print-module-deps universal/*.jar quarkus-run.jar 
```

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.6.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.6.0))
